engine/src/main/resources/kettle-job-entries.xml

```diff
@@ -13,7 +13,7 @@
  <job-entry id="SFTPPUT"> <description>i18n:org.pentaho.di.job.entry:JobEntry.SFTPPut.TypeDesc</description> <classname>org.pentaho.di.job.entries.sftpput.JobEntrySFTPPUT</classname> <category>i18n:org.pentaho.di.job:JobCategory.Category.FileTransfer</category> <tooltip>i18n:org.pentaho.di.job.entry:JobEntry.SFTPPut.Tooltip</tooltip> <iconfile>ui/images/PSFTP.svg</iconfile> <documentation_url>http://wiki.pentaho.com/display/EAI/Put+a+file+with+SFTP</documentation_url> <cases_url/> <forum_url/> </job-entry>
  <job-entry id="FTP_DELETE"> <description>i18n:org.pentaho.di.job.entry:JobEntry.FTPDELETE.TypeDesc</description> <classname>org.pentaho.di.job.entries.ftpdelete.JobEntryFTPDelete</classname> <category>i18n:org.pentaho.di.job:JobCategory.Category.FileTransfer</category> <tooltip>i18n:org.pentaho.di.job.entry:JobEntry.FTPDELETE.Tooltip</tooltip> <iconfile>ui/images/FTPD.svg</iconfile> <documentation_url>http://wiki.pentaho.com/display/EAI/FTP+Delete</documentation_url> <cases_url/> <forum_url/> </job-entry>
  <job-entry id="TABLE_EXISTS"> <description>i18n:org.pentaho.di.job.entry:JobEntry.TableExists.TypeDesc</description> <classname>org.pentaho.di.job.entries.tableexists.JobEntryTableExists</classname> <category>i18n:org.pentaho.di.job:JobCategory.Category.Conditions</category> <tooltip>i18n:org.pentaho.di.job.entry:JobEntry.TableExists.Tooltip</tooltip> <iconfile>ui/images/TEX.svg</iconfile> <documentation_url>http://wiki.pentaho.com/display/EAI/Table+Exists+%28Job+Entry%29</documentation_url> <cases_url/> <forum_url/> </job-entry>
- <job-entry id="FILE_EXISTS"> <description>i18n:org.pentaho.di.job.entry:JobEntry.FileExists.TypeDesc</description> <classname>org.pentaho.di.job.entries.fileexists.JobEntryFileExists</classname> <category>i18n:org.pentaho.di.job:JobCategory.Category.Conditions</category> <tooltip>i18n:org.pentaho.di.job.entry:JobEntry.FileExists.Tooltip</tooltip> <iconfile>ui/images/FEX.svg</iconfile> <documentation_url>Products/File_Exists</documentation_url> <cases_url/> <forum_url/> </job-entry>
+ <job-entry id="FILE_EXISTS"> <description>i18n:org.pentaho.di.job.entry:JobEntry.FileExists.TypeDesc</description> <classname>org.pentaho.di.job.entries.fileexists.JobEntryFileExists</classname> <category>i18n:org.pentaho.di.job:JobCategory.Category.Conditions</category> <tooltip>i18n:org.pentaho.di.job.entry:JobEntry.FileExists.Tooltip</tooltip> <iconfile>ui/images/FEX.svg</iconfile> <documentation_url>Products/File_Exists_(Job_Entry)</documentation_url> <cases_url/> <forum_url/> </job-entry>
```

Comment:
Should this use url encodings like line 15 %28 and %29 for the ( ) ?

Reply:
@cseverino789 No, that is not necessary.